src/sentry/south_migrations/0492_auto__add_field_groupredirect_orgnaization_id__del_unique_groupredirec.py

```diff
@@ -0,0 +1,1468 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import SchemaMigration
+from django.db import models
+
+
+class Migration(SchemaMigration):
+
+    # Flag to indicate if this migration is too risky
+    # to run online and needs to be coordinated for offline
+    is_dangerous = False
+
+    def forwards(self, orm):
+        # Adding field 'GroupRedirect.organization_id'
+        db.add_column('sentry_groupredirect', 'organization_id',
+                      self.gf('sentry.db.models.fields.bounded.BoundedBigIntegerField')(null=True),
+                      keep_default=False)
+
+        db.commit_transaction()
+        db.execute(
+            "CREATE UNIQUE INDEX CONCURRENTLY {} ON sentry_groupredirect (organization_id, previous_short_id, previous_project_slug)".
+            format(
+                db.create_index_name(
+                    'sentry_groupredirect', [
+                        'organization_id', 'previous_short_id', 'previous_project_slug']),
+            )
+        )
+        db.start_transaction()
+
+        # Removing unique constraint on 'GroupRedirect', fields
+        # ['previous_short_id', 'previous_project_slug']
+        db.delete_unique(u'sentry_groupredirect', ['previous_short_id', 'previous_project_slug'])
+
+        # Adding unique constraint on 'GroupRedirect', fields ['organization_id',
+        # 'previous_short_id', 'previous_project_slug']
+        db.create_unique(
+            'sentry_groupredirect', [
+                'organization_id', 'previous_short_id', 'previous_project_slug'])
+
+    def backwards(self, orm):
+        # Removing unique constraint on 'GroupRedirect', fields
+        # ['organization_id', 'previous_short_id', 'previous_project_slug']
+        db.delete_unique(
+            'sentry_groupredirect', [
+                'organization_id', 'previous_short_id', 'previous_project_slug'])
+
+        # Deleting field 'GroupRedirect.organization_id'
+        db.delete_column('sentry_groupredirect', 'organization_id')
+
+        # Adding unique constraint on 'GroupRedirect', fields
+        # ['previous_short_id', 'previous_project_slug']
+        db.commit_transaction()
+        db.execute(
+            "CREATE UNIQUE INDEX CONCURRENTLY {} ON sentry_groupredirect (previous_short_id, previous_project_slug)".
+            format(
+                db.create_index_name(
+                    'sentry_groupredirect', [
+                        'previous_short_id', 'previous_project_slug']),
+            )
+        )
+        db.start_transaction()
```

Comment:
Should there be another migration to backfill the `organization_id` column?

Reply:
yeah from Slack I mentioned it'd be pretty easy to do, and will follow up once the DDL is in